doc/v0.3.0/conf.py

```diff
@@ -0,0 +1,339 @@
+# -*- coding: utf-8 -*-
```

Comment:
better make the layout as
doc/conf.py
doc/latest
doc/v1.0.0
doc/v0.3.0

Then all version can share the pages for community/development/conf.py/etc.